react-native-sdk/update_dependencies.js

```diff
@@ -32,14 +44,9 @@ function updateDependencies() {
             return item;
         }, {});
 
-    console.log(
-        'Updating dependencies:',
-        Object.keys(packageJSON.dependencies)
-    );
-
     const data = JSON.stringify(packageJSON, null, 2);
 
-    fs.writeFileSync('../../package.json', data);
+    fs.writeFileSync(packageJSON, data);
```

Comment:
This is not a path but an object.

Reply:
So, this should be fs.writeFileSync(path.resolve(__dirname, '../../package.json'), data).